lib/WUI/wui_api.cpp

```diff
@@ -353,6 +353,14 @@ uint32_t wui_gcodes_uploaded() {
     return uploaded_gcodes;
 }
 
+static void set_current_file(char *filename) {
+    strlcpy(marlin_vars()->media_LFN, basename(filename), FILE_NAME_BUFFER_LEN);
```

Comment:
What is the purpose here? I mean, I'm setting this only for my copy of the buffer, right?

I'd like to point out that the filename isn't guaranteed to be LFN. Actually, most of the time, it _isn't_, most of the time it's a SFN path. There's a rare occasion (starting print just after an upload) where it might be LFN and that's why it is explicitly always "shortened" in the original code.

If we want this to be LFN, we should convert it to LFN (there's a function for that direction too).